examples/credential_request_sd_jwt_vc.json

```
@@ -0,0 +1,10 @@
+{
+   "format": "vc+sd-jwt",
+   "credential_definition": {
+      "vct": "IdentityCredential"
+   },
+   "proof": {
+      "proof_type": "jwt",
+      "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpLXByb29mK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwiY3J2IjoiUC0yNTYiLCJ4Ijoick00bmJvTmlrVm1LQlB2dVd3Q0NTeDRYTFBLNlZjUEp5NDlYOEJyd0Q2OCIsInkiOiI1cXhkZ1hvVVh1RFJPMGVxZjlvWHhKLUhzeDM2QlVDOXNoeWtxNk9NRU9RIn19.eyJhdWQiOiJodHRwczovL2lzc3Vlci1vcGVuaWQ0dmMuc3NpLnRpci5idWRydS5kZSIsImlhdCI6MTcwMTk2MDQ0NCwibm9uY2UiOiJMYXJSR1NibVVQWXRSWU82QlE0eW44In0.lOqcNuzNacLwyKqIy5zBK2BQ05flHr6dIGgvXT0IHQC06VBVgT1dI6MjwwsDjs3c6-M_aWfDAFltLya6UKIW_Q"
```

The JWT contains the wrong `aud`. It should be `https://credential-issuer.example.com` to make sense in the example.

---

This suggestion fixes the aud:

```suggestion
      "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpLXByb29mK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwiY3J2IjoiUC0yNTYiLCJ4IjoiblVXQW9BdjNYWml0aDhFN2kxOU9kYXhPTFlGT3dNLVoyRXVNMDJUaXJUNCIsInkiOiJIc2tIVThCalVpMVU5WHFpN1N3bWo4Z3dBS18weGtjRGpFV183MVNvc0VZIn19.eyJhdWQiOiJodHRwczovL2NyZWRlbnRpYWwtaXNzdWVyLmV4YW1wbGUuY29tIiwiaWF0IjoxNzAxOTYwNDQ0LCJub25jZSI6IkxhclJHU2JtVVBZdFJZTzZCUTR5bjgifQ.-a3EDsxClUB4O3LeDD5DVGEnNMT01FCQW4P6-2-BNBqc_Zxf0Qw4CWayLEpqkAomlkLb9zioZoipdP-jvh1WlA"
```